lib/braveAlerter.js

```diff
@@ -289,7 +299,29 @@ class BraveAlerter {
     } else {
       const errorMessage = `Bad request to ${request.path}: ${validationErrors.array()}`
       helpers.logError(errorMessage)
-      response.status(400).send(errorMessage)
+      response.status(400).json(errorMessage)
+    }
+  }
+
+  async handleGetActiveAlerts(request, response) {
+    const validationErrors = Validator.validationResult(request).formatWith(helpers.formatExpressValidationErrors)
+
+    if (validationErrors.isEmpty()) {
+      const alertApiKey = request.header('X-API-KEY')
+
+      const activeAlerts = await this.getActiveAlertsByAlertApiKey(alertApiKey)
+      if (activeAlerts === null) {
+        const errorMessage = `Something went wrong in request to ${request.path}. Active Alerts is null.`
+        helpers.logError(errorMessage)
+        response.status(500).json(errorMessage)
```

Comment:
so, does this error message just get displayed where the alerts would otherwise be?

Reply:
No, there is a separate `ErrorScreen` that appears and will display any error messages that are returned by API calls.